alpha/scripts/utils/updatePartnerEntries2Sphinx.php

```diff
@@ -51,8 +51,16 @@
 		if ($mode == 'execute')
 		{
 			echo "Updating [". $item->getId() ."] ... "; 
-			$sphinx->saveToSphinx($item, false, true);
-			echo "Done\n";
+			try {
+				$sphinx->saveToSphinx($item, false, true);
+				echo 'Done' . PHP_EOL;
+			} catch (PDOException $e) {
+				if (strpos($e->getMessage(), 'unknown column') !== false) {
```

Comment:
php8 introduced 'str_contains' which is more readable:
str_contains($e->getMessage(), 'unknown column')